pkgs/games/srb2kart/default.nix

```
@@ -50,39 +73,46 @@ in stdenv.mkDerivation rec {
   ];
 
   cmakeFlags = [
-    #"-DSRB2_ASSET_DIRECTORY=/build/source/assets"
+    "-DSRB2_ASSET_DIRECTORY=${finalAttrs.assets}/share/srb2kart"
     "-DGME_INCLUDE_DIR=${game-music-emu}/include"
     "-DSDL2_MIXER_INCLUDE_DIR=${lib.getDev SDL2_mixer}/include/SDL2"
     "-DSDL2_INCLUDE_DIR=${lib.getDev SDL2}/include/SDL2"
   ];
 
-  patches = [
-    ./wadlocation.patch
+  # Desktop icon
+  desktopItems = [
+    (makeDesktopItem rec {
+      name = "Sonic Robo Blast 2 Kart";
+      exec = finalAttrs.pname;
+      icon = finalAttrs.pname;
+      comment = finalAttrs.meta.description;
```

```suggestion
      comment = meta.description;
```
updating meta should not trigger rebuilds

---

Can you explain? I can't remove `finalAttrs` since stdenv doesn't use `rec`

---

```suggestion
      comment = "kart racing mod based on Sonic Robo Blast 2";
```
kinda duplicatd with meta.description, maybe we find something shorter or more fitting for the desktop icon but so we avoid triggering rebuilds when meta is changed

---

Oh I understand now, thanks. I think that comment is good and abbreviated the last part to SRB2.